test/membrane_element_rtp_aac/pipeline_test.exs

```diff
@@ -0,0 +1,33 @@
+defmodule Membrane.RTP.AAC.Pipeline.Test do
+  use ExUnit.Case, async: true
+
+  import Membrane.ChildrenSpec
+  import Membrane.Testing.Assertions
+
+  defp prepare_test_payload(_ctx) do
+    %{
+      payload: [<<1>>]
+    }
+  end
+
+  setup :prepare_test_payload
+
+  test "payloader to depayloader does not change source", ctx do
+    spec = [
+      child(:source, %Membrane.Testing.Source{output: ctx.payload, stream_format: %Membrane.AAC{}})
+      |> child(:payloader, Membrane.RTP.AAC.Payloader)
+      |> child(:depayloader, Membrane.RTP.AAC.Depayloader)
+      |> child(:sink, Membrane.Testing.Sink)
+    ]
+
+    {:ok, _supervisor_pid, pipeline_pid} = Membrane.Testing.Pipeline.start(spec: spec)
```

Comment:
[NIT] I would make sure the pipeline is linked to the test process, as well as I would use the functions with `!` at the end of the function name :D 
```suggestion
    pipeline_pid = Membrane.Testing.Pipeline.start_link_supervised!(spec: spec)
```